External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -226,7 +236,11 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 %% ====================================================================================
 %% 3. Create new NIfTI
 if ~isempty(changeMat)
-	newNifti.mat = changeMat;
+	newNifti.mat  = changeMat;
+end
+
+if ~isempty(changeMat0)
+	newNifti.mat0 = changeMat0;

```

Comment:
Please add a warning here that you change mat0

warning('Note that mat0 (the original NIfTI orientation) is changed upon request, this can jeopardize re-re-running ExploreASL!');

Because when we rerun ExploreASL, several orientations are defaulted back to mat0. So in this case, you need this warning. If you don't need it for import, then we need a bVerbose boolean to suppress such warnings.

But if this is a very exceptional case, then we can just always issue the warning.

Reply:
It's a very exceptional case and the user can be aware of that. So I'd indeed always issue this warning without a possibility to suppress. 

Warning added.